scripts/image-builder/configure.sh

```diff
@@ -17,6 +17,9 @@ sudo dnf install -y "https://dl.fedoraproject.org/pub/epel/epel-release-latest-$
 sudo dnf install -y mock caddy tomcli
 sudo usermod -a -G mock "$(whoami)"
 
+# Set the Correct Permissions for osbuild-composer
+[ "${HOME_PERM}" -lt 0711 ]  && chmod 0711 ~
```

Comment:
I'd rather if we did delta changes instead of absolute changes. Who knows what other permissions someone might need for other reasons?

```suggestion
[ "${HOME_PERM}" -lt 0711 ]  && chmod go+x ~
```